assignment5/index.html

```diff
@@ -0,0 +1,43 @@
+    </ul>
+  </nav>
+  <section id="home">
+    <h1>Welcome home!</h1>
+    <p>
+      Bacon ipsum dolor amet ham bacon turducken, pastrami beef meatloaf biltong prosciutto short ribs jerky doner t-bone capicola jowl fatback. Biltong beef ribs pastrami, picanha filet mignon pork loin pig tri-tip frankfurter cupim. Beef venison meatloaf tri-tip, ground round pork belly pig. Shankle fatback tenderloin, flank burgdoggen short loin turkey beef ball tip. Pork belly turducken jowl beef burgdoggen swine brisket venison. Burgdoggen pork loin porchetta, shankle corned beef salami landjaeger prosciutto turkey swine turducken tri-tip.
+    </p>
+    <p>
+      Pig picanha turducken doner bresaola chuck shank. Turducken kevin burgdoggen alcatra landjaeger, meatball flank ribeye bacon pork loin kielbasa. Kevin bresaola andouille corned beef cupim. Kevin jowl salami, swine tail short ribs bacon.
+    </p>
+    <p>
+      Ground round jowl pancetta beef ribs. Rump chuck tail alcatra ham kevin turkey tenderloin porchetta pastrami pig cupim tongue beef ribs. Bresaola t-bone short ribs shankle ribeye ground round shoulder pork loin turkey. Ham hock beef ribs turducken, frankfurter cow picanha short ribs ball tip capicola turkey tenderloin cupim pork loin bacon. Frankfurter venison pork loin boudin jerky, short loin ground round kevin tri-tip ham hock. Meatball venison pig frankfurter strip steak tri-tip pork loin corned beef.
+    </p>
+  </section>
+  <section id="best-practices">
+    <h1>Beste practices for Front-end code</h1>
+    <p>
+      Bacon ipsum dolor amet ham bacon turducken, pastrami beef meatloaf biltong prosciutto short ribs jerky doner t-bone capicola jowl fatback. Biltong beef ribs pastrami, picanha filet mignon pork loin pig tri-tip frankfurter cupim. Beef venison meatloaf tri-tip, ground round pork belly pig. Shankle fatback tenderloin, flank burgdoggen short loin turkey beef ball tip. Pork belly turducken jowl beef burgdoggen swine brisket venison. Burgdoggen pork loin porchetta, shankle corned beef salami landjaeger prosciutto turkey swine turducken tri-tip.
+    </p>
+    <p>
+      Pig picanha turducken doner bresaola chuck shank. Turducken kevin burgdoggen alcatra landjaeger, meatball flank ribeye bacon pork loin kielbasa. Kevin bresaola andouille corned beef cupim. Kevin jowl salami, swine tail short ribs bacon.
+    </p>
+    <p>
+      Ground round jowl pancetta beef ribs. Rump chuck tail alcatra ham kevin turkey tenderloin porchetta pastrami pig cupim tongue beef ribs. Bresaola t-bone short ribs shankle ribeye ground round shoulder pork loin turkey. Ham hock beef ribs turducken, frankfurter cow picanha short ribs ball tip capicola turkey tenderloin cupim pork loin bacon. Frankfurter venison pork loin boudin jerky, short loin ground round kevin tri-tip ham hock. Meatball venison pig frankfurter strip steak tri-tip pork loin corned beef.
+    </p>
+  </section>
+</body>
+<script src="./static/app.js"></script>
```

Comment:
Script tag outside of body [won't validate](https://validator.w3.org/nu/?doc=https%3A%2F%2Fiancstewart.github.io%2Fwafs%2F).